manager/gui/src/lib/components/SgtToggle/SgtToggle.vue

```diff
@@ -58,8 +61,13 @@ export default class SgtToggle extends Vue {
   opacity: 0.5;
   pointer-events: none;
 }
-.toggle-wrapper .label {
-  margin-right: 5px;
+.toggle-wrapper{
+  .labelPre {
```

Comment:
Try to avoid writing extra CSS for padding and margin.
Used Vuetify Predefined classes like mr-2 and ml-2

Reply:
Done.